src/ColorPicker/ColorPicker.csproj

```diff
@@ -3,15 +3,15 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I think we should update it to .NET 5 since PixiEditor is the main target, however, we can make alternative packages for .NET Core and .NET framework if possible.